smithy-aws-iam-traits/.gitignore

```diff
@@ -0,0 +1,3 @@
+gradle/
```

Comment:
Can remove this, the top level `.gitignore` should handle these.

Reply:
done